ui/Screen/Project/Source.svelte

```diff
@@ -221,8 +238,8 @@
         {#if object.info.objectType === ObjectType.Blob}
           <FileSource
             blob={object}
-            path={$currentPath}
-            rootPath={path.projectSource(project.id)}
+            path={currentObjectPath}
+            rootPath={path.projectSource(project.id, project.default_user.id)}
```

Comment:
So I'm guessing this PR doesn't scope files to the currently selected peer? Just commits?

Reply:
Indeed, changing the remaining endpoints to accept a `userId` (or `peerId`) is still TBD. Wanted to get the UI routing merged first before @xla rewrites the whole router...

Reply:
The remaining endpoints are taken care of in:
https://github.com/radicle-dev/radicle-upstream/pull/620/commits/e2c4a7f0d339a145ca4c94b8b18cd57116b897f8
https://github.com/radicle-dev/radicle-upstream/pull/620/commits/d4f044e705b391b371409455a496b9941fedb075
https://github.com/radicle-dev/radicle-upstream/pull/620/commits/f25eb586bf32e7ddc3abd70fce13cdd2466af24c
https://github.com/radicle-dev/radicle-upstream/pull/620/commits/f25eb586bf32e7ddc3abd70fce13cdd2466af24c